experiments/20161104-pol-assay-high-throughput/20161104-first-attempt.md

```diff
@@ -0,0 +1,18 @@
+## Goals
+- Plan out the adaption of pol assay protocol to 96-well plate
+
+### New Equipment
+- a multi-channel aspirator is in the works, but for now I will use a multi-channel pipette to efficiently remove medium from wells
+
+### Plate Seeding
+
+How long to achieve 80% confluence? To start with, I will try seeding ~140K cells per well with the expectation, based on previous experiments with 12-well plates, that the cell count will quadruple in one day to 560K. ~80% confluence for a 12-well plate represented about 4.5 million cells per well, and we are estimating the area of a 96-well plate well to be 1/8 that of a 12-well plate well.
+
+### Transfection
+
+| Reagent                   | 12-well Volume (ul/well) | 96-well Volume (ul/well) | Dilution |
+|---------------------------|--------------------------|--------------------------|----------|
+| Plasmids                  | 1                        | 1                        | 1:8      |
+| PHH                       | 1.5                      | 1.5                      | 1:8      |
+| pCD                       | 0.67                     | 0.67                     | 1:8      |
+| OPTIMEM + Lipofectamine MM | 252.5                    | 31.56                    | 1:1      |
```

Comment:
May want to note after the table that the concentrations in the 96-well plate volumes will result in reagents 8x as concentrated as they are in the 12-well plate. (that's the right calculation, right?)